### PR TITLE
Nora - Feb 2020

### PR DESCRIPTION
**Pull request by noravdh**

### CODE REVIEW

**Review by xie-lang — COMMENTED** [review]
Is it necessary to give filtered results a section separate from the main message board ?

**Review by xie-lang — COMMENTED** [review]
Are there any tests for the Class DatabaseConnection?

File: lib/peep.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  def self.filter(filter:)
+
+    result = DatabaseConnection.query("SELECT * FROM peeps WHERE message LIKE '%#{filter}%' ORDER BY timestamp DESC;")
+    @filtered_messages = result.map do |peep| 
+      Peep.new(message: peep['message'], timestamp: peep['timestamp']) 
+    end
+  end
+
+  def self.filtered_messages
```
Comment (lindseydew) [inline lib/peep.rb]: I don't think you should need an instance variable here. The data could just be returned from the filter method

File: spec/features/see_date_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+feature 'View list of peeps' do
+  scenario 'user visits the homepage' do
+    visit '/'
+
+    expect(page).to have_content "2020-03-13"
```
Comment (lindseydew) [inline spec/features/see_date_spec.rb]: Nice!

File: spec/peep_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  it "can filter peeps" do
+    messages = Peep.filter(filter: "second")
+
+    expect(messages.length).to eq 1
+    expect(messages.last).not_to eq "This is a peep!"
+  end
+
+end
```
Comment (lindseydew) [inline spec/peep_spec.rb]: Excellent tests

File: views/filter.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+<body>
+<div class='parent'>
+  <div class='child inline-block-child-1'><a href="/"><img src='rubber-duck.png'></a></div>
+  <div class='child inline-block-child-1'><h1>Welcome to Chitter!</h1></div>
+</div>
+<div class='parent'>
+<div class='child inline-block-child'>
+
+<h2>Filtered Peeps</h2>
+
```
Comment (lindseydew) [inline views/filter.erb]: Lovely styling!

**Review by lindseydew — COMMENTED** [review]
Amazing job on this! ⭐️ You've implemented all the functionality with tests - great job 💯

**Pull request closed without merging**